plugins/@grouparoo/app-templates/src/destination/templates.ts

```diff
@@ -1,7 +1,11 @@
-import { ConfigTemplate } from "@grouparoo/core";
+import { ConfigTemplate, DestinationSyncMode } from "@grouparoo/core";
 
 export class DestinationTemplate extends ConfigTemplate {
-  constructor(name: string, files: string[]) {
+  constructor(
+    name: string,
+    files: string[],
+    syncModes?: DestinationSyncMode[]
```

Comment:
You had mentioned you might want to leave `["sync"]` as the default.   I think an empty array is better - it's clear that this isn't implemented yet.